### ISSUE TITLE
No background in Arthur's house

### ISSUE THREAD

**Issue by andwhyisit** [andwn/cave-story-md#35]
Compare the bottom of the screen for reference:
http://i214.photobucket.com/albums/cc164/andwhyisit/arthurshouse_zpsj0fpldnv.gif

Also the flowers and flower entity seem to be missing.

While I'm here, though it is entirely cosmetic, it might be worth removing the leading 0 from the health bar.

Tested in v0.2.0-alpha

**Comment by andwn** [comment]
I've added the background but I'm not too happy with the bright color of it. Will have to try and take care of that later. In your other issue also Graveyard had a black background, so I removed the background from that map.

Noticed after adding in the sprite that the red flower entity was raised for some reason, so that's why you couldn't interact with it.

HUD should show just the single digit now too.
